### ISSUE TITLE
Navigating to a page in the navbar breaks after first time

### ISSUE THREAD

**Issue by Dbrrohof** [FlutterFlow/flutterflow-issues#3900]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

If you have the following pages in the navbar:
Homepage
Search page

Then you have a widget on the homepage that navigates to the search page. On the search page you click the navbar to go back to the homepage. Trying to click the widget with the navigation to the search page again does not work.

### Expected Behavior

When clicking an element with a "Navigate to" action, it should navigate to that page even after the first time.

### Steps to Reproduce

1. Create two pages, Homepage + Page B
2. Add a widget to the homepage with a "Navigate to" action to page b.
3. Disable "Allow back navigation"
4. Now run the app
5. Click the widget
6. Click the homepage in the navbar
7. Click the widget on the homepage again
8. Observe that nothing is happening

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITESls/l6odgrtQB7of2YMFqhR4jNlx7bZNEiMF+GA02F5zyEJwpWv/DTBdpZsO8d25mGlWiik0B+dL8hezpKfcFGAyHb7pI1qpIbg7dcGW6MLqyEcy0SHNtJ/NTfnaF4KaNpx1RLbFtdV1pig/GSw==

### Visual documentation

https://github.com/user-attachments/assets/f07da215-264e-4071-9e48-d3427e4d3542

### Environment

```markdown
- FlutterFlow version: v5.0.0 
- Platform: Windows
- Browser name and version: Chrome, 129.0.6668.70 
- Operating system and version affected: Windows 11 Home, 23H2
```

**Comment by matchia-ai** [comment]
For me as well, when I disable the “Allow Back Navigation,” the page I’m navigating to does not refresh with the new parameters I’m passing in the second time. For example, if I have dynamic child components that each navigate to the same page with different parameters, on the second navigation, the page will not refresh with the new parameters if “Allow Back Navigation” is disabled. when it enabled the issue gone, but i dont want it enabled.

**Comment by Dbrrohof** [comment]
Don't close it.

**Comment by Dbrrohof** [comment]
bump

**Comment by Dbrrohof** [comment]
bump

**Comment by Dbrrohof** [comment]
bump

**Comment by Dbrrohof** [comment]
bump

**Comment by Dbrrohof** [comment]
bump

**Comment by Dbrrohof** [comment]
Any news on this since its still a bug???

**Comment by Alezanello** [comment]
Hi, we sincerely apologize for not catching this issue earlier. It appears there was a temporary sync issue between GitHub and our internal tool, which caused some reports to be missed. We're now aware of it and will be working on it right away

**Comment by Alezanello** [comment]
Hi again!

Just a quick explanation regarding the "Allow back navigation" setting (let's call it ANB for short):

When ANB is disabled, FlutterFlow uses context.goNamed() for navigation. This method replaces the current route or reuses an existing one if it's already in the navigation stack. So, if you're navigating to PageB this way, and then return via the bottom navigation bar (which also uses goNamed), the home page isn’t fully recreated. When you tap the same navigation again, it might not do anything—because it thinks PageB is already in the stack.

How to fix it:
Enable ANB. This switches the navigation method to context.pushNamed(), which always pushes a new instance of PageB onto the stack. That ensures the transition happens every time, avoiding the “stuck” behavior you noticed.

In short:
✅ Enable Allow back navigation in the Navigate To action to ensure consistent navigation behavior.
